app/views/posts/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<section class="container">
+<p class="post-title"> Create New Post :</p>
+<%= form_with model: @post, url: user_posts_path do |form| %>
+     <div>
+     <%= form.label :title %> <br>
+     <%= form.text_field :title %>
+     </div>
+
+     <div>
+     <%= form.label :text %> <br>
+     <%= form.text_field :text %>
```

Comment:
- [ ] [OPTIONAL] Good job so far 👏 I think you could use the `Textarea` input so as to give the users a large space to write in 😃 How does that sound?

Reply:
Thank you @BertrandConxy  i will implement this right away.